src/jit/hwintrinsicxarch.cpp

```diff
@@ -265,6 +408,15 @@ bool Compiler::compSupportsHWIntrinsic(InstructionSet isa)
                                                     isFullyImplmentedISAClass(isa));
 }
 
+static bool is64bitOnlyIntrinsicOn32bitPlatform(HWIntrinsicCategory category, var_types type)
```

Comment:
I think @CarolEidt suggested this be named differently above (might've gotten hidden by GitHub)